app/mailers/unread_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UnreadMailer < ApplicationMailer
+  def notice(sender, reciever, chat_url)
```

Comment:
__[Misspell]__

"reciever" is a misspelling of "receiver"


<!-- Comment by [Sider](https://sider.review) -->